src/components/widgets/PresentationItem.js

```diff
@@ -70,7 +65,7 @@ const styles = () => ({
  * @returns {*}
  * @constructor
  */
-const PresentationItem = ({ label, helpText, required, classes, children, width }) => {
```

Comment:
We still have this parameter in the description and in the list of PropTypes. By the way, why you decided to get rid of it?

Reply:
The thinking was that it wasn't interesting to know what fields are mandatory, when you aren't editing. Just clutter in that case. Do you find that it was useful?